x/accounts/internal/implementation/api_builder.go

```diff
@@ -0,0 +1,106 @@
+package implementation
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+var (
+	errNoInitHandler    = errors.New("no init handler")
+	errNoExecuteHandler = errors.New("account does not accept messages")
+	ErrInvalidMessage   = errors.New("invalid message")
```

Comment:
These are under internal, so is there any sense in making any of them public?

Reply:
no this is a mistake, I will make it private